experimenter/experimenter/experiments/constants.py

```diff
@@ -994,6 +994,45 @@ class FirefoxLabsGroups(models.TextChoices):
 Optional - We believe this outcome will <describe impact> on <core metric>
     """  # noqa
 
+    KPI_AREA = "KPI Metrics"
+    NOTABLE_CHANGES_AREA = "Notable Changes"
+    DEFAULT_METRIC_AREAS = [NOTABLE_CHANGES_AREA, KPI_AREA]
+
+    DAILY_ACTIVE_USERS = "client_level_daily_active_users_v2"
+    DAYS_OF_USE = "days_of_use"
+    RETENTION = "retained"
+    SEARCH_COUNT = "search_count"
```

Comment:
These are defined in an enum in the Jetstream schemas already if you want to pull them from there: https://github.com/mozilla/experimenter/blob/80fef74d645aa670678956d6ee4127f16f561072/experimenter/experimenter/jetstream/models.py#L29-L34